### ISSUE TITLE
Allow for multiple dom-modules

### ISSUE THREAD

**Issue by bryandcoulter** [webpack-contrib/polymer-webpack-loader#12]
Currently the loader looks for a single dom-module in the incoming source. The loader need to be able to process multiple dom-modules.

**Comment by ChadKillingsworth** [comment]
After mulling over this a bit, there's a few concerns we should handle at the same time.

 * We should recognize only root level `<dom-module>` elements. If they are nested somewhere, we should ignore them.
 * We also need to recognize other root level elements and use the `toBody` register method on them.
 * I think we should only transform script elements outside of `<template>` elements.

```html
<dom-module></dom-module> <!-- Normal - register this one -->
<template> <!-- This should be recognized and registered with toBody -->
  <dom-module></dom-module> <!-- Nothing special should be done for this dom-module -->
</template>

<dom-module>
  <template>
    <script></script> <!-- Don't touch this script - it's in a template - this is weird -->
  </template>
  <script></script> <!-- normal script -->
</dom-module>
```

**Comment by ChadKillingsworth** [comment]
Oh and we should only call `parse5.parse` once per file. Parsing and serializing is an expensive operation.

**Comment by ChadKillingsworth** [comment]
This was fixed by #51
